### PR TITLE
0405 split i18n files

### PR DESCRIPTION
**Pull request by zmstone**
Fixes [EMQX-9487](https://emqx.atlassian.net/browse/EMQX-9487)

### Split the i18n files into per-language.
Now `rel/i18n/*.hocon` holds all the English descriptions, and `rel/i18n/zh/*.hocon` holds the zh translations.
The next step is to move zh to a separate repository and should be managed by writers.

### Changed the way we cache i18n descriptions.
The cache loads in all the languages which are found by listing the `schema.<lang>.hocon` files.
This means we are now ready for more languages extracted from dashboard package.

### Swagger spec multi-lang support is made dynamic.
Now one can evaluate this expression: `emqx_conf:update([dashboard], {change_i18n_lang, zh}, #{}).`
to change the swagger spec doc language, no need to restart the node.

### The hot-config and bridges schema JSON dumps are now made dynamic.
Instead of static files generated at build time, now the schemas can be retrieved from this API:
```
curl -X 'GET' 'http://zmd:18083/api/v5/schemas/hotconf?lang=zh'
```

### Changed schema dump scripts to work with split files
For now, we'll still keep dumping the `docgen` artifacts as before.
Two types of files can be eliminated later:
1. A new way to present the schema, then we can stop dumping the markdown docs
2. When the dashboard starts retrieving the schema JSONs from the new API, we can delete the old files, namely hot-config-schema-{en,zh}.json and bridges-api-{en,zh}.json.

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [x] Added tests for the changes
- [x] Changed lines covered in coverage report
- [~] Change log has been added to `changes/{ce,ee}/(feat|perf|fix)-<PR-id>.en.md` files
- [x] For internal contributor: there is a jira ticket to track this change
- [~] If there should be document changes, a PR to emqx-docs.git is sent, or a jira ticket is created to follow up
- [~] Schema changes are backward compatible

## Checklist for CI (.github/workflows) changes

- [~] If changed package build workflow, pass [this action](https://github.com/emqx/emqx/actions/workflows/build_packages.yaml) (manual trigger)
- [~] Change log has been added to `changes/` dir for user-facing artifacts update

### CODE REVIEW

File: apps/emqx_dashboard/src/emqx_dashboard_desc_cache.erl [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+%% all the desc.<lang>.hocon files in the www/static directory (extracted from dashboard package).
+init() ->
+    ok = ensure_app_loaded(emqx_dashboard),
+    PrivDir = code:priv_dir(emqx_dashboard),
+    EngDesc = filename:join([PrivDir, "desc.en.hocon"]),
+    WwwStaticDir = filename:join([PrivDir, "www", "static"]),
+    OtherLangDesc0 = filelib:wildcard("desc.*.hocon", WwwStaticDir),
+    OtherLangDesc = lists:map(fun(F) -> filename:join([WwwStaticDir, F]) end, OtherLangDesc0),
+    Files = [EngDesc | OtherLangDesc],
+    ?MODULE = ets:new(?MODULE, [named_table, public, set, {read_concurrency, true}]),
```
Comment (thalesmg) [inline apps/emqx_dashboard/src/emqx_dashboard_desc_cache.erl:39]: Maybe:

```suggestion
    ?MODULE = emqx_utils_ets:new(?MODULE, [public, ordered_set, {read_concurrency, true}]),
```
?

File: apps/emqx_dashboard/src/emqx_dashboard_listener.erl [hunk 2]
```diff
@@ -68,6 +68,9 @@ handle_call(_Request, _From, State) ->
 handle_cast(_Request, State) ->
     {noreply, State, hibernate}.
 
+handle_info(i18n_lang_changed, _State) ->
+    NewState = regenerate_minirest_dispatch(),
```
Comment (thalesmg) [inline apps/emqx_dashboard/src/emqx_dashboard_listener.erl:72]: This can only be `ready` now, right?

**Review by thalesmg — APPROVED** [review]

File: apps/emqx_dashboard/src/emqx_dashboard_listener.erl (hunk 2 above)
Comment (zmstone) [reply apps/emqx_dashboard/src/emqx_dashboard_listener.erl:72]: yes.

**Review by zhongwencool — APPROVED** [review]

File: apps/emqx_dashboard/src/emqx_dashboard_desc_cache.erl (hunk 1 above)
Comment (zmstone) [reply apps/emqx_dashboard/src/emqx_dashboard_desc_cache.erl:39]: will fix in a follow up pr.

**Pull request merged**